src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ConnectionPool/HttpConnectionPool.cs

```diff
@@ -689,9 +709,10 @@ private async ValueTask<Stream> ConnectToTcpHostAsync(string host, int port, Htt
             }
             catch (Exception ex)
             {
+                activity?.Stop();
```

Comment:
we'll need to record `error.type`

Reply:
This PR doesn't add attributes and it would be weird to record only `error.type`. We are tracking the addition of attributes to the request activity in #93019. We may consider adding attributes to the connection span, but I would do it in a separate PR together after resolving #93019. @samsp-msft do you think think this would be valuable? Let's open a separate tracking issue if yes.

Reply:
sounds good!